docs/content/preview/yugabyte-cloud/cloud-basics/create-clusters/include-general-settings.md

```diff
@@ -11,5 +11,5 @@ private = true
 Set the following options:
 
 - **Cluster Name**: Enter a name for the cluster.
-- **Provider**: Choose a cloud provider - AWS or GCP.
+- **Provider**: Choose a cloud provider - AWS, Azure (single region only), or GCP.
```

Comment:
Can remove "single region only" from here as user can't create multi-region clusters in the UI/API anyways. We can just mention it in the limitations section.